LSP.sublime-settings

```diff
@@ -7,6 +7,9 @@
 
   // --- Document Formatting ------------------------------------------------------------
 
+  // Run the server's documentRangeFormattingProvider (if supported) on the pasted text.
+  "format_on_paste": false,
```

Comment:
Worth to make it a global setting (`lsp_format_on_paste`) maybe? Then it's at least possible to set it per syntax and per project (like `lsp_format_on_save`) which might be useful given how different servers can handle range formatting better or worse.